examples/animation/blend-trees-2d-cartesian.html

```diff
@@ -0,0 +1,220 @@
+                        },
+                    ]
+                },
+            ],
+            "parameters": {
+                "posX": {
+                    "name": "posX",
+                    "type": "FLOAT",
+                    "value": 0
+                },
+                "posY": {
+                    "name": "posY",
+                    "type": "FLOAT",
+                    "value": 0
+                }
+            }
+        };
+
+        var canvas = document.getElementById("application-canvas");
+
+        // Create the application and start the update loop
+        var app = new pc.Application(canvas);
+
+        // Set the canvas to fill the window and automatically change resolution to be the same as the canvas size
+        app.setCanvasFillMode(pc.FILLMODE_FILL_WINDOW);
+        app.setCanvasResolution(pc.RESOLUTION_AUTO);
+
+        window.addEventListener("resize", function () {
+            app.resizeCanvas(canvas.width, canvas.height);
+        });
+
+        var miniStats = new pcx.MiniStats(app);
+
+        var entity;
+
+        // Create an Entity with a camera component
+        var cameraEntity = new pc.Entity(app);
+        cameraEntity.addComponent("camera", {
+            clearColor: new pc.Color(0.1, 0.15, 0.2)
+        });
+        cameraEntity.translateLocal(0.0, 0.75, 5.0);
+        app.root.addChild(cameraEntity);
+
+        app.scene.ambientLight = new pc.Color(0.5, 0.5, 0.5);
+
+        // Create an entity with a light component
+        var light = new pc.Entity();
+        light.addComponent("light", {
+            type: "directional",
+            color: new pc.Color(1, 1, 1),
+        });
+        light.setLocalEulerAngles(45, 30, 0);
+        app.root.addChild(light);
+
+        // Load model and animation assets
+        var model = null;
+        var idleAnim = null;
+        var danceAnim = null;
+        var walkAnim = null;
+
+        var stateGraphAsset = new pc.Asset('stateGraph', 'animstategraph', null, animStateGraph);
+        stateGraphAsset.once('load', function (asset) {
+            var stateGraph = asset.resource;
+        app.assets.loadFromUrl("../assets/models/bitmoji.glb", "container", function (err, asset) {
+            model = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/idle.glb", "container", function (err, asset) {
+            idleAnim = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/dance.glb", "container", function (err, asset) {
+            danceAnim = asset;
+        app.assets.loadFromUrl("../assets/animations/bitmoji/walk.glb", "container", function (err, asset) {
+            walkAnim = asset;
+
+            var modelEntity = model.resource.instantiateRenderEntity({
+                    castShadows: true
+            });
+
+            modelEntity.addComponent('anim', {
+                activate: true,
+            });
+
+            modelEntity.anim.loadStateGraph(stateGraph);
```

Comment:
it'd be good to add explanations, especially to anim related code, as it acts as tutorial for people